### ISSUE TITLE
Clinet OS

### ISSUE THREAD

**Issue by IamDontNoob** [Wutze/OpenVPN-WebAdmin#44]
Is there is way to find from what OpenVPN client (Androin, iOS, Linux, Windows, Mac) user connected? In Pritunl it is very conveniently displayed and very informative.
Could you add a user client display to the control panel?

**Comment by Wutze** [comment]
I never thought this information was important because I don't want to spy on users. Sure, it is possible to filter this information out of the logs, but is the effort justified?

However, I will save this as a wish. Maybe there will be another point of view ;o)

**Comment by IamDontNoob** [comment]
Yes, I understand you, and that it will take a certain amount of time. But maybe you can do it for a donation?
I need this in order to understand some things, since I do it in a country where a lot is prohibited, as well as some protocols, and when they tell me that something does not work, I need to know from which client they are connecting, so that do not ask unnecessary questions. If you remember, I created a question about how to change the encryption protocol from tls-auth to tls-crypt, this protocol does not work in turkmenistan on certain devices and i have to replace it with tls-crypt. some users just can't answer this question because they don't understand it.

**Comment by Wutze** [comment]
All right, I can understand that though and that actually even makes sense. I hadn't considered that there are also people who are denied a really free internet.

I'm also taking a closer look at how I can solve this problem. Thank you for the advice.

**Comment by Wutze** [comment]
A first solution would be to use "multitail" on the shell.

I have just pushed a corresponding version into the git, with which the operating system and the client version used are highlighted in red. (Documentation in the file)

Provided that a log file exists.

https://github.com/Wutze/OpenVPN-WebAdmin/blob/master/installation/snippets/multitail.conf

**Comment by IamDontNoob** [comment]
Sorry, but how can i install this update and check client os?

**Comment by IamDontNoob** [comment]
Ah, understood, found it in server log.
Would by nice if it's can be added in Webadmin

**Comment by Wutze** [comment]
I will do that as soon as I find the time ;o)

**Comment by Wutze** [comment]
I have found a solution and am working on it at the moment. More later.

**Comment by IamDontNoob** [comment]
Wow, thank you so much!!!

**Comment by IamDontNoob** [comment]
Hello, any news?

**Comment by Wutze** [comment]
Yes, I had planned to finish it at the weekend. Unfortunately, my job is getting on my nerves a bit because a lot of people are on holiday or sick at the moment. That's why I can't give any exact information about it yet. Sorry.
